src/Npgsql/TypeHandlers/NetworkHandlers/InetHandler.cs

```diff
@@ -36,23 +41,27 @@ public override IPAddress Read(NpgsqlReadBuffer buf, int len, FieldDescription f
             [CanBeNull] FieldDescription fieldDescription,
             bool isCidrHandler)
         {
-            buf.ReadByte();  // addressFamily
+            buf.ReadByte(); // addressFamily
             var mask = buf.ReadByte();
             var isCidr = buf.ReadByte() == 1;
             Debug.Assert(isCidrHandler == isCidr);
             var numBytes = buf.ReadByte();
             var bytes = new byte[numBytes];
-            for (var i = 0; i < numBytes; i++) {
+            for (var i = 0; i < numBytes; i++)
+            {
                 bytes[i] = buf.ReadByte();
             }
```

Comment:
Curly brackets. Do you really need them here?

Reply:
Nope!